geniza/corpus/metadata_export.py

```diff
@@ -135,11 +136,16 @@ def get_export_data_dict(self, doc):
         outd["doc_date_original"] = doc.doc_date_original
         outd["doc_date_calendar"] = doc.get_doc_date_calendar_display()
         outd["doc_date_standard"] = doc.doc_date_standard
-        outd["notes"] = doc.notes
-        outd["needs_review"] = doc.needs_review
-        outd[
-            "url_admin"
-        ] = f"{self.url_scheme}{self.site_domain}/admin/corpus/document/{doc.id}/change/"
+
+        if not "notes" in self.exclude_csv_fields:
+            outd["notes"] = doc.notes
+        if not "needs_review" in self.exclude_csv_fields:
+            outd["needs_review"] = doc.needs_review
+
+        if not "url_admin" in self.exclude_csv_fields:
+            outd[
+                "url_admin"
+            ] = f"{self.url_scheme}{self.site_domain}/admin/corpus/document/{doc.id}/change/"
```

Comment:
I think the the admin doc exporter should extend `get_export_data_dict` and add these fields, rather than adding conditional behavior in the base class